service/controller/v25/resource/tccp/spec.go

```diff
@@ -2,20 +2,6 @@ package tccp
 
 import "github.com/aws/aws-sdk-go/service/cloudformation"
 
-const (
-	// defaultCreationTimeout is the timeout in minutes for the creation of the
-	// stack.
-	defaultCreationTimeout = 20
-
-	workerRoleKey = "WorkerRole"
-
-	namedIAMCapability = "CAPABILITY_NAMED_IAM"
-
-	// versionBundleVersionParameterKey is the key name of the Cloud Formation
-	// parameter that sets the version bundle version.
-	versionBundleVersionParameterKey = "VersionBundleVersionParameter"
-)
-
 // StackState is the state representation on which the resource methods work.
 type StackState struct {
```

Comment:
I plan to phase out `StackState` completely and go with the default resource interface in the future just like with all the other cloud formation stack resources. We are going to simplify all of this. Note that this resource implementation here was one of the very first we ever implemented. We advanced a bit since then.